src/mpyl/project.py

```diff
@@ -127,6 +127,20 @@ def from_config(values: dict):
         )
 
 
+@dataclass(frozen=True)
+class KeyId:
```

Comment:
why `KeyId` and not `Credentials` or something?

Reply:
Sure